### PR TITLE
[WIP] make gem compatible with latest solidus

### PR DESCRIPTION
**Pull request by ccarruitero**
This is an initial attempt to get the gem compatible with more solidus versions

The upload is working, but some specs aren't passing.

What do you think?

### CODE REVIEW

File: app/models/spree/product_import.rb [hunk 1]
```diff
@@ -1,602 +1,620 @@
...
-        else
-          priceStr=priceStr.gsub(',', '')
-        end
-      end
-      #We replace comma by dot.
-      return priceStr.gsub(',', '.').to_f
-    end
-  end
-end
+# This model is the master routine for uploading products
```
Comment (ccarruitero) [inline app/models/spree/product_import.rb]: changing the file format here makes this difficult to review.

I'm going to move this to another PR.

**Comment by ahwong5** [comment]
@ccarruitero does these fixes works for Solidus v2.3? Thanks.

**Comment by NoelDiazMesa** [comment]
this is a nice job, we want to review and acept this. Thanks a lot!!

**Comment by ccarruitero** [comment]
@ahwong5 I think this should work with 2.3

@NoelDiazMesa I just removed the commit changing the file format, so you can review the changes more easier. But we should fix the file formats in the near future, looking the `^M` at line endings make me feel weird
